cluster-autoscaler/cloudprovider/azure/azure_scale_set_test.go

```diff
@@ -138,12 +136,9 @@ func TestIncreaseSize(t *testing.T) {
 	err := provider.azureManager.forceRefresh()
 	assert.NoError(t, err)
 
-	ss := newTestScaleSet(provider.azureManager, "test-asg")
-	ss.lastSizeRefresh = time.Now()
-	ss.sizeRefreshPeriod = 1 * time.Minute
-	ss.curSize = -1
+	ss := newTestScaleSet(provider.azureManager, "test-asg-doesnt-exist")
 	err = ss.IncreaseSize(100)
-	expectedErr := fmt.Errorf("the scale set test-asg is under initialization, skipping IncreaseSize")
+	expectedErr := fmt.Errorf("could not find vmss: test-asg-doesnt-exist")
```

Comment:
technically with the changes in #3717, we'll never end with the "-1" initialization case.